clidocstool_md.go

```diff
@@ -153,6 +153,9 @@ func mdCmdOutput(cmd *cobra.Command, old string) (string, error) {
 		fmt.Fprint(b, "\n\n")
 	}
 
+	// necessary to merge persistent flags
```

Comment:
This is a bit confusing, because the description of `Flags()` says (is that a bug in Cobra?);

```
// Flags returns the complete FlagSet that applies
// to this command (local and persistent declared here and by all parents).
```

Wondering if we need to use a different function below (instead of `Flags()`) ?

Reply:
> is that a bug in Cobra?

Might be linked to https://github.com/spf13/cobra/issues/412